### DISCUSSION TITLE
add root permission ？

### DISCUSSION THREAD

**Discussion by liuhouer** [rustdesk/rustdesk#9848] [General]
Can the Android version add root permission application? After giving root permission, you can wake up the phone by turning off the screen. There are currently no restrictions on accessibility and settings in the background, and it will automatically stop after a while. As a result, other devices cannot be woken up, and the Android device cannot be connected and controlled. The phone must be manually unlocked and woken up.

----------------------------
安卓版可否加入root权限申请？给了root权限后，可以息屏唤醒手机。现在的无障碍和设置后台不限制，过一会还是会自动停止。导致其他设备唤不醒，无法连接控制这个安卓设备，必须手动再把手机解锁唤醒才行。

**Comment by Poordeveloper** [comment]
It is open source, nobody forbid you add it. I guess no if you only want to wait for other's job.
